platform/commonUI/browse/test/PaneControllerSpec.js

```diff
@@ -59,6 +63,13 @@ define(
                     ["isMobile", "isPhone", "isTablet", "isPortrait", "isLandscape"]
                 );
                 mockWindow = jasmine.createSpyObj("$window", ["open"]);
+
+                mockLocation = { search: function () {
```

Comment:
For consistency, can you change this to a spy?  And then you don't have to redefine mockLocation down on line 103; you only need to change the return value.